src/vm/excep.cpp

```diff
@@ -7556,20 +7551,22 @@ void InitSavedExceptionInfo()
 void FaultingExceptionFrame::Init(CONTEXT *pContext)
 {
     WRAPPER_NO_CONTRACT;
-#if defined(_TARGET_X86_)
+#ifndef WIN64EXCEPTIONS
+#ifdef _TARGET_X86_
     CalleeSavedRegisters *pRegs = GetCalleeSavedRegisters();
     pRegs->ebp = pContext->Ebp;
     pRegs->ebx = pContext->Ebx;
     pRegs->esi = pContext->Esi;
     pRegs->edi = pContext->Edi;
     m_ReturnAddress = ::GetIP(pContext);
     m_Esp = (DWORD)GetSP(pContext);
-#elif defined(WIN64EXCEPTIONS)
+#else // _TARGET_X86_
+    PORTABILITY_ASSERT("FaultingExceptionFrame::InitAndLink");
```

Comment:
The assert message is stale, could you please fix it since you are touching this code?